### PR TITLE
Testimonial section updated.

### PR DESCRIPTION
**Pull request by Pragati-3003**
I have updated the testimonial section of the website.
Issues#59

### CODE REVIEW

**Comment by praveenscience** [comment]
Use the right PR template, thanks.

File: css/style.css [hunk 1]
```diff
@@ -1379,25 +1379,55 @@ select.form-control.has-value {
...
+  transform: scale(0.98);
+  box-shadow: rgba(17, 17, 26, 0.1) 0px 1px 0px;
+  background-color: white;
+ }
 /*owl carousel slider*/
 .owl-carousel .owl-item .client-img img {
   width: 60px;
   display: initial;
+  
 }
```
Comment (praveenscience) [inline css/style.css:1431]: Use prettier.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Please use the proper PR format. Use prettier to make the code look right.

File: css/style.css (hunk 1 above)
Comment (Pragati-3003) [reply css/style.css:1431]: Okie, I'll format code using prettier.

**Comment by Pragati-3003** [comment]
> Use the right PR template, thanks.

Okie

**Comment by praveenscience** [comment]
Waiting for the updated one. 😊

**Comment by Pragati-3003** [comment]
> Waiting for the updated one. 😊

Updated

**Review by praveenscience — APPROVED** [review]
👍

**Comment by Pragati-3003** [comment]
> 👍

Thanku for merge my PR 😊. And please add level number :)

**Pull request merged**